content/rdi/_index.md

```diff
@@ -40,6 +42,7 @@ Redis Data Integration supports the following database sources using [Debezium S
 | SQL Server         | 2017, 2019             |
 | Cassandra          | >= 3.0                 |
 | Datastax Cassandra | >= 6.8.0               |
+| MongoDB            | 4.2, 4.4, 5.0, 6.0  driver: 4.7 |
```

Comment:
This list was originally alphabetically sorted, keep it that way.

Reply:
done